bundles/org.smarthomej.binding.tr064/src/main/java/org/smarthomej/binding/tr064/internal/Tr064RootHandler.java

```diff
@@ -423,6 +442,15 @@ public String getFriendlyName() {
 
     @Override
     public Collection<Class<? extends ThingHandlerService>> getServices() {
-        return Set.of(Tr064DiscoveryService.class, PhonebookActions.class);
+        return Set.of(Tr064DiscoveryService.class, PhonebookActions.class, FritzboxActions.class);
```

Comment:
Should we maybe distinguish between handler for `generic` Thing and `fritzbix` thing in the future?

Reply:
done